x-pack/plugins/security_solution/public/timelines/store/timeline/reducer.ts

```diff
@@ -520,6 +522,26 @@ export const timelineReducer = reducerWithInitialState(initialTimelineState)
       },
     },
   }))
+  .case(initializeSavedSearch, (state, { id, savedSearch }) => ({
+    ...state,
+    timelineById: {
+      ...state.timelineById,
```

Comment:
hey, check out `immer` - we have in our codebase. it is a nice way to avoid doing spreads like these:) just fyi

Reply:
I wish, haha. There's actually a refactoring planned in which we migrate the timeline redux code to redux toolkit which comes with immer. <3